README.md

```diff
@@ -396,13 +396,14 @@ Finally, we can use the `make kind-run` target to run the current version of kub
 Every time you want to test a change, you'll need to rebuild the docker image and push it to cluster before running it again. ( `make build-docker kind-push kind-run` )
 
 ## Contributing
+Please encourage and support all the contribution to this project. Kindly read [Contributing.md](CONTRIBUTING.md) before contributing. Some instructions for the common contributions are stated below.
```

Comment:
```suggestion
Kindly read [Contributing.md](CONTRIBUTING.md) before contributing. Some instructions for the common contributions are stated below.
```

Deleted the first sentence as it's not clear who it's aimed at (contributors or maintainers?) I do like the addition of a link to the Contributing page.